setup.py

```diff
@@ -33,7 +33,7 @@
     install_requires=[
         "PyStemmer>=1.3,<2.0",
         "bblfsh>=2.2.1,<3.0",
-        "modelforge>=0.9.3,<0.10",
+        "modelforge>=0.10.1,<0.11",
```

Comment:
`modelforge 0.11` is not included - is it correct?

Reply:
Oh modelforge 0.11 was just relesed yesterday, let me update it